contents/blog/send-love-to-open-source.md

```diff
@@ -0,0 +1,57 @@
+
+> **For every GitHub star we receive on [the PostHog repo](https://github.com/PostHog/posthog) between now through to the end Valentine's Day, we'll donate $5 USD across the four handpicked open-source projects.**
+
+We'll donate a maximum of $50,000 to these projects in total. 
+
+## Who gets the money? 
+
+We're donating to the following projects, all of which are either used extensively in our website or product, or have been suggested for their daily usefulness to our team outside of work.
+
+| **Project**                                                    | **What is it?                                                     | **Why are we supporting it?**                                                                 |
+|----------------------------------------------------------------|-------------------------------------------------------------------|-----------------------------------------------------------------------------------------------|
+| [Rrweb](https://github.com/rrweb-io/rrweb)                     | An open-source web session replay library                         | Rrweb powers our [Session Recording](https://posthog.com/docs/user-guides/recordings) feature |
+| [Django REST](https://github.com/encode/django-rest-framework) | A powerful toolkit for building Web APIs | [explanation here]                                                                            |
+| [Signal](https://github.com/signalapp)                         | Open-source private messenger for iOS, Android and Mac            | All the PostHog team are frequent users                                                       |
+| [Homebrew](https://github.com/Homebrew/)                       | Package manager for Mac                                           | [explanation here]                                                                            |
+
+
+## How does this work?
+
+That's easy. Between now and the end of Valentine's Day, simply follow these steps:
+
+1. Login to GitHub and go to [the PostHog repo](https://github.com/PostHog/posthog).
+2. Click the ⭐️ icon. Hint: it's in the top right corner.
+3. That's it. We'll donate on your behalf!
+
+On the morning of Tuesday, February 15, we'll count up how many stars we've gained and contact the projects above to make the donations. We'll quickly verify the stars are legitimate (see <em>small print</em> below).
+
+## Why are we doing this?
+
+PostHog is an open-source project and our team is proud to be part of the open-source community. We wanted to take this week as an opportunity to truly celebrate and support the open-source community in a meaningful way.
+
+We also wanted to involve our community, which is why we've chosen to give donations based on GitHub stars. If you'd like to unstar us after we've counted your contributions, that's absolutely fine - we suggest just waiting until later in the week to be sure your star is counted towards the total.
```

Comment:
```suggestion
We also wanted to involve our community, which is why we've chosen to give donations based on GitHub stars. If you'd like to unstar us after we've counted your contributions, that's absolutely fine - we suggest just waiting until later in the week to be sure your star is counted towards the total.
```

The real reason is the paragraph below. This paragraph feels odd - it's counter to our aim below, and takes us away from the aim of the exercise (which makes it harder to justify in future). Perhaps remove?